utils/oci-runtime-tools/Makefile

```
@@ -0,0 +1,72 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oci-runtime-tools
+PKG_VERSION:=0.9.0
+PKG_RELEASE:=1
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_SOURCE_NAME:=runtime-tools
+PKG_SOURCE:=$(PKG_SOURCE_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/opencontainers/runtime-tools/archive/v$(PKG_VERSION)
+PKG_SOURCE_VERSION:=ee63cfa6f66491e78f8e63c37806d7a905eb836e
+PKG_HASH:=4c2978e9097a3c93648c8ecf4f785f728044cddf717686e0b05a9244a1ad273c
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_SOURCE_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
```

What about something like this?  It's compile tested, and it's simpler and cleaner.

```suggestion
PKG_NAME:=oci-runtime-tools
PKG_VERSION:=0.9.0
PKG_RELEASE:=1

PKG_SOURCE:=v$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://github.com/opencontainers/runtime-tools/archive/v$(PKG_VERSION)
PKG_HASH:=4c2978e9097a3c93648c8ecf4f785f728044cddf717686e0b05a9244a1ad273c
PKG_BUILD_DIR:=$(BUILD_DIR)/runtime-tools-$(PKG_VERSION)

PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
PKG_LICENSE:=Apache-2.0
PKG_LICENSE_FILES:=LICENSE
```

---

That PKG_SOURCE is no good. It potentially conflicts with other packages sharing the same version. Codeload would be better as it redirects to it anyway.